demos/crossroad_camera_demo/main.cpp

```diff
@@ -253,17 +253,22 @@ struct PersonAttribsDetection : BaseDetection {
     std::string outputNameForAttributes;
     std::string outputNameForTopColorPoint;
     std::string outputNameForBottomColorPoint;
+    bool hasTopBottomColor;
 
 
-    PersonAttribsDetection() : BaseDetection(FLAGS_m_pa, "Person Attributes Recognition") {}
+    PersonAttribsDetection() : BaseDetection(FLAGS_m_pa, "Person Attributes Recognition"), hasTopBottomColor(false) {}
 
     struct AttributesAndColorPoints{
         std::vector<std::string> attributes_strings;
         std::vector<bool> attributes_indicators;
+
+        bool hasTopBottomColor;
```

Comment:
This value can't change from one instance of `AttributesAndColorPoints` to another. This is a fixed property of `PersonAttribsDetection` for the whole run. May be it's better to introduce a function for `PersonAttribsDetection` signalling if it predicts TopBottomColor.

Reply:
Agree. I will change this.

Reply:
Since we keep `hasTopBottomColor` in `PersonAttribsDetection` I think it will be enough to check this instead of having a function.

Reply:
Fixed